Gemfile.lock

```diff
@@ -1,7 +1,6 @@
 GIT
-  remote: git://github.com/lightguard/aweplug.git
-  revision: ecd6d369ff1cc61bac7295af73e2f3f9d7bfcc94
-  branch: vimeo_connection_reset_fix
+  remote: git://github.com/awestruct/aweplug.git
+  revision: 95554705783f74bad302a00033457f7f7f4dd968
```

Comment:
I think you meant to move to this commit: https://github.com/LightGuard/aweplug/commit/ecd6d369ff1cc61bac7295af73e2f3f9d7bfcc94, which doesn't seem to be in: https://github.com/awestruct/aweplug/commits/master